components/calendar/src/persian.rs

```diff
@@ -614,18 +542,6 @@ mod tests {
         }
     }
 
-    #[test]
```

Comment:
Is there a reason why we don't preserve this test?

Reply:
the function it is testing no longer exists and the replacement function (persian_from_fixed) is already tested below